### PR TITLE
moves user endpoint to webapp. cleans up service route string instances

### PR DESCRIPTION
**Pull request by esotrope**

### CODE REVIEW

File: webapp/src/main/java/org/opentestsystem/rdw/reporting/user/UserContextController.java [hunk 1]
```diff
@@ -3,9 +3,11 @@
 import org.opentestsystem.rdw.reporting.common.web.security.User;
 import org.springframework.security.core.annotation.AuthenticationPrincipal;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
+@RequestMapping("/api/user")
 class UserContextController {
```
Comment (esotrope) [inline webapp/src/main/java/org/opentestsystem/rdw/reporting/user/UserContextController.java]: moved to webapp

File: webapp/src/main/webapp/src/app/admin/groups/groups.service.ts [hunk 2]
```diff
@@ -9,8 +9,9 @@ import { URLSearchParams } from "@angular/http";
 import { Group } from "./model/group.model";
 import { DataService } from "../../shared/data/data.service";
 import { map } from 'rxjs/operators';
+import { AdminServiceRoute } from '../../shared/service-route';
 
-const ServiceRoute = '/admin-service';
+const ServiceRoute = AdminServiceRoute;
```
Comment (esotrope) [inline webapp/src/main/webapp/src/app/admin/groups/groups.service.ts]: unrelated cleanup - replaced service route string instances with references to a central service route constants

File: webapp/src/test/java/org/opentestsystem/rdw/reporting/user/UserContextControllerIT.java [hunk 3]
```diff
@@ -39,11 +39,12 @@ public void itShouldReturnWhatServiceReturns() throws Exception {
                         .build()
         );
 
-        mvc.perform(get("/user"))
+        mvc.perform(get("/api/user"))
```
Comment (malaffoon) [inline webapp/src/test/java/org/opentestsystem/rdw/reporting/user/UserContextControllerIT.java]: Will this effectively change this end-point back to what it was in v1.0? It was "reporting-host/api/user" in v1.0, "reporting-host/api/reporting-service/user" in v1.1, and now will be "reporting-host/api/user" again?

Comment (esotrope) [reply webapp/src/test/java/org/opentestsystem/rdw/reporting/user/UserContextControllerIT.java]: correct 👍

File: webapp/src/main/webapp/src/app/groups/results/group-assessment.service.ts [hunk 4]
```diff
@@ -4,11 +4,12 @@ import { AssessmentExamMapper } from "../../assessments/assessment-exam.mapper";
 import { ExamFilterOptionsService } from "../../assessments/filters/exam-filters/exam-filter-options.service";
 import { AssessmentProvider } from "../../assessments/assessment-provider.interface";
 import { ResponseUtils } from "../../shared/response-utils";
-import { Group } from "../../groups/group";
+import { Group } from "../group";
```
Comment (malaffoon) [inline webapp/src/main/webapp/src/app/groups/results/group-assessment.service.ts]: don't see this change elsewhere in this PR, was it just wrong?

Comment (esotrope) [reply webapp/src/main/webapp/src/app/groups/results/group-assessment.service.ts]: intellij optimization. there were two ways to get to the same file and this was the shorter

Comment (malaffoon) [reply webapp/src/main/webapp/src/app/groups/results/group-assessment.service.ts]: oic, i missed the double `../`
